applications/node.vechain.energy.md

```diff
@@ -0,0 +1,147 @@
+# VeChain General Grant Application Template
+
+## Project Overview
+
+- Project: Upgrade and Deployment of public Nodes
+- Team Name: favo
+- Payment Address: `0xcC4B3412161Ea88d0538D2Da15cffa74af0eE9D4`
+
```

Comment:
Hi @ifavo, I noticed that the application file in this pull request (#147) is the same as the one in pull request #115. Could you please help remove this file and just keep the delivery files?